package-lock.json

```diff
@@ -1,10534 +1,8 @@
 {
   "name": "@splunk/otel",
   "version": "0.12.0",
-  "lockfileVersion": 2,
```

Comment:
Was it on purpose to downgrade the lockfile version?

Reply:
It was not. Fixing.